src/client/components/TabPaneContent3/index.js

```diff
@@ -81,9 +76,8 @@ function TabPaneContent3() {
         <Trans t={t} i18nKey="tab3-p7">
           We understand that it’s a difficult time to look for a job right now.
           You are <strong>currently not</strong> required to upload your resume
-          to{" "}
-          <a href="https://www.caljobs.ca.gov/vosnet/Default.aspx">CalJOBS</a>,
-          during the COVID-19 pandemic.
+          to <a href={t("links.caljobs")}>CalJOBS</a>, during the COVID-19
+          pandemic.
```

Comment:
This isn't showing up correctly in the snapshot for some reason.

Reply:
Yeah, it looks like it's not showing up in the UI either.

Reply:
Huh. Somehow the node numbering changed from 4 to 3. I guess the URL string counted as a text node and the t() does not count as a node at all? i18next is such a mystery...

Thanks!